example/components/SlidingTabNavigationExample.js

```diff
@@ -17,6 +19,18 @@ export default class SlidingTabNavigationExample extends Component {
     },
   }
 
+  _goToFirstTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('first');
+    });
+  };
+
+  _goToSecondTab = () => {
+    this.props.navigation.performAction(({ tabs, stacks }) => {
+      tabs('sliding-tab-navigation').jumpToTab('second');
```

Comment:
yeah +1 for that, @brentvatne that's the issue I was telling you about yesterday.

Also it seems like `SlidingTabNavigation` from exponent takes `navigationState` as a prop, but it is not used... /cc: @skevy